src/WebJobs.Script.Grpc/Abstractions/WorkerDescription.cs

```diff
@@ -1,28 +1,41 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
 namespace Microsoft.Azure.WebJobs.Script.Abstractions
 {
     public class WorkerDescription
     {
         /// <summary>
         /// Gets or sets the name of the supported language. This is the same name as the IConfiguration section for the worker.
         /// </summary>
+        [JsonProperty(PropertyName = "language", Required = Required.Always)]
         public string Language { get; set; }
 
         /// <summary>
         /// Gets or sets the supported file extension type. Functions are registered with workers based on extension.
         /// </summary>
+        [JsonProperty(PropertyName = "extension", Required = Required.Always)]
         public string Extension { get; set; }
```

Comment:
Should this be an array? E.g. for Node in v1, we supported both .js and typescript https://github.com/Azure/azure-functions-host/blob/1e38fafa4410cc60720c1a680f3b104244b6a473/src/WebJobs.Script/Description/Node/Compilation/JavaScriptCompilationServiceFactory.cs#L17

Reply:
Will address this a separate PR. Issue https://github.com/Azure/azure-functions-host/issues/2984